lib/syntax_tools/src/erl_syntax.erl

```diff
@@ -246,6 +246,9 @@
 	 macro/2,
 	 macro_arguments/1,
 	 macro_name/1,
+         map_comp/2,
+         map_comp_template/1,
+         map_comp_body/1,
```

Comment:
Actually not. The new lines are indented with spaces (as they should be), but the `macro_name/1` line is still indented with a tab because it's untouched.